program: |
    extern fn bar(a: int, b: bool);
    fn main() : int {
      bar(4);
      return 0;
    }
    
expect_compile_failure: true
expect_stderr: |
  3:3 Type error: 2 arguments expected but 1 provided
    bar(4);
    ^^^
  1:11 Note: Function declared here
  extern fn bar(a: int, b: bool);
            ^^^
  Terminating compilation due to 1 error